### ISSUE TITLE
intelmqctl start without --group option exit with error

### ISSUE THREAD

**Issue by 0xffca** [certtools/intelmq#1253]
Fresh install of intelmq on ubuntu 14.04 

intelmqctl start starts the botnet but exit with the following error.
If I specify the group I want to start there is no error 
```
Traceback (most recent call last):
  File "/usr/local/bin/intelmqctl", line 11, in <module>
    load_entry_point('intelmq==1.1.0a1', 'console_scripts', 'intelmqctl')()
  File "/usr/local/lib/python3.4/dist-packages/intelmq/bin/intelmqctl.py", line 1154, in main
    return x.run()
  File "/usr/local/lib/python3.4/dist-packages/intelmq/bin/intelmqctl.py", line 593, in run
    retval, results = args.func(**args_dict)
  File "/usr/local/lib/python3.4/dist-packages/intelmq/bin/intelmqctl.py", line 604, in bot_start
    return self.botnet_start(group=group)
  File "/usr/local/lib/python3.4/dist-packages/intelmq/bin/intelmqctl.py", line 687, in botnet_start
    log_botnet_message('running', group)
  File "/usr/local/lib/python3.4/dist-packages/intelmq/bin/intelmqctl.py", line 98, in log_botnet_message
    logger.info(MESSAGES[status], BOT_GROUP[group] + (" group" if group else ""))
KeyError: None
```

**Comment by wagner-certat** [comment]
Thanks for the report. I pushed a quick fix with a86bc3d8

The bug has been introduced in ca613152d, the code looks like it should have already covered this case, but it didn't.

cc @e3rd

**Comment by e3rd** [comment]
Thank you @wagner-certat and sorry that I overlooked this before!
